integration-tests/src/test/resources/features/AddressAPIHappyPath.feature

```diff
@@ -26,9 +27,9 @@ Feature: Address API happy path test
     #Credential Issue
     When user sends a POST request to Credential Issue end point with a valid access token
     And a valid JWT is returned in the response
-    And JWT lives for two hours
-
-    #Get_Addresses
-    When user sends a GET request to Addresses end point
-    Then user should receive a valid response
 
+    Examples:
+      | testUserDataSheetRowNumber | testPostCode |
+      | 197                        | SW1A 2AA     |
```

Comment:
What extra value are we getting from having multiple scenarios here?

Reply:
@gokhakan would you like to comment on this please?

Reply:
Done.